layouts/index.html

```diff
@@ -119,27 +119,27 @@ <h1 class="font-sans font-body font-normal">
                     <img class="mx-auto lg:mx-8 h-12 mx-8" src="/logos/customers/tableau_logo.png" alt="Tableau Software" title="Tableau Software">
                 </a>
                 <a data-track="customer-logo" href="{{ relref . "/case-studies/mercedes-benz" }}">
-                    <img class="mx-auto lg:mx-8 mt-8 mb-4 lg:my-0 h-12 mx-8" src="/logos/customers/mercedes-benz-RDNA_logo.png" title="Mercedes-Benz Research and Development" alt="Mercedes-Benz Research and Development">
+                    <img class="mx-auto lg:mx-8 mt-8 mb-4 lg:my-0 h-12 mx-8" src="/logos/customers/logo-section/mercedes-benz-RDNA_logo.png" title="Mercedes-Benz Research and Development" alt="Mercedes-Benz Research and Development">
```

Comment:
Maybe submit these changes as a separate PR, since they're unrelated?

Reply:
They're related because the images are a bit big in the logo section and Google Page Speed was complaining. I didn't want to edit the main logo images to avoid messing up any other place they're used. Though I can break it out into two PRs.

Reply:
Were you planning on doing this? (Breaking it into a separate PR, I mean.)

I will say that if the images really are are "big", since they're PNGs and of a fixed size, so you should just optimize them in place, and not create copies of them. If you grep for them you should find anywhere else they're referenced. But I suspect it should be fine to do this.